openhabian-setup.sh

```diff
@@ -59,7 +59,7 @@ OLDWD=$(pwd) && cd /opt
 if [[ -n "$UNATTENDED" ]]; then
   # apt/dpkg commands will not try interactive dialogs
   export DEBIAN_FRONTEND=noninteractive
-  apt-get update
+  apt-get update >/dev/null
```

Comment:
hm, does a cond_... make sense here?

Reply:
yes but I simply undid this here because it is to change anyway in #724